dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -1,6 +1,33 @@
-{% macro dbt_trino_get_append_sql(tmp_relation, target_relation) %}
+{% macro dbt_trino_validate_get_incremental_strategy(config) %}
+  {#-- Find and validate the incremental strategy #}
+  {%- set strategy = config.get("incremental_strategy", default="append") -%}
+
+  {% set invalid_strategy_msg -%}
+    Invalid incremental strategy provided: {{ strategy }}
+    Expected one of: 'append', 'delete+insert'
+  {%- endset %}
+  {% if strategy not in ['append', 'delete+insert'] %}
+    {% do exceptions.raise_compiler_error(invalid_strategy_msg) %}
+  {% endif %}
+
+  {% do return(strategy) %}
+{% endmacro %}
+
+{% macro dbt_trino_get_incremental_sql(strategy, tmp_relation, target_relation, unique_key, dest_columns) %}
+  {% if strategy == 'append' %}
+    {% do return(get_append_sql(target_relation, tmp_relation, dest_columns)) %}
+  {% elif strategy == 'delete+insert' %}
+    {% do return(get_delete_insert_merge_sql(target_relation, tmp_relation, unique_key, dest_columns)) %}
+  {% elif strategy == 'merge' %}
+    {#-- Not yet supported by trino --#}
+    {% do return(get_merge_sql(target_relation, tmp_relation, unique_key, dest_columns)) %}
```

Comment:
Is this a valid strategy for Trino at the moment?

https://github.com/trinodb/trino/issues/7708

Reply:
Note that this strategy is currently disabled:

```
  {% set invalid_strategy_msg -%}
    Invalid incremental strategy provided: {{ strategy }}
    Expected one of: 'append', 'delete+insert'
  {%- endset %}
  {% if strategy not in ['append', 'delete+insert'] %}
    {% do exceptions.raise_compiler_error(invalid_strategy_msg) %}
  {% endif %}
```